src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/base/GT_MetaTileEntity_MultiblockBase_EM.java

```diff
@@ -3210,4 +3212,14 @@ public void draw(float partialTicks) {
     }
 
     // endregion
+
+    public void turnOffMaintenance() {
```

Comment:
Why should this function and the variable `isNeededMaintenanceChecks` be public?